composer.json

```diff
@@ -5,7 +5,8 @@
     "require": {
         "php": "^5.6|^7.0",
         "aws/aws-sdk-php": "^3.0",
-        "illuminate/support": "^5.0|^6.0"
+        "illuminate/support": "^5.0|^6.0",
+        "google/cloud-kms": "^1.7"
```

Comment:
I wonder if we should drop both the aws and google dependencies from here, and instead add instructions in the README alongside each driver. Might be better for users to install only the one they need for the driver they intend to use? Thoughts?

Reply:
You may be onto something. That thought had occurred to me as well.

Reply:
Yeah, I went ahead and just merged it anyway. It's not a big issue... yet. I'll wait and see how many future drivers are added, split these out in a future release if needed.